python/neuroglancer/lib.py

```diff
@@ -46,6 +55,25 @@ def helper(elem, shape, i):
 
     return helper(elem, shape, 0)
 
+
+def find_closest_divisor(to_divide, closest_to):
```

Comment:
Have faith! The use case is coming in a future PR. I think indirect tests might be too let me check....